frontend/internal-packages/agent/src/chat/workflow/types.ts

```diff
@@ -30,6 +30,10 @@ export type WorkflowState = {
   ddlExecutionFailed?: boolean | undefined
   ddlExecutionFailureReason?: string | undefined
 
+  // DML execution results
+  dmlExecutionSuccessful?: boolean | undefined
```

Comment:
nits

I wonder if dmlExecutionSuccess might be a bit more concise than dmlExecutionSuccessful. What do you think?